src/tools/gravity-parseList.c

```diff
@@ -219,13 +199,6 @@ int gravity_parseList(const char *infile, const char *outfile, const char *adlis
 		{
 			// ABP-style match (see comments above)
 
-			// Skip lines containing ABP extended CSS selectors
-			// ("##", "#!#", "#@#", "#?#") preceded by a letter
-			// See https://github.com/pi-hole/pi-hole/pull/5247 for
-			// further information on why this is necessary
-			if(regexec(&abp_css_regex, line, 1, &match, 0) == 0)
```

Comment:
If this is removed we should also remove the corresponding definition (`#define ...`) and the variable + compilation of the regex.